cranelift/codegen/src/isa/x64/inst.isle

```diff
@@ -801,6 +802,8 @@
             NLE
             P
             NP))
+(decl from_intcc (IntCC) CC)
+(extern constructor from_intcc from_intcc)
```

Comment:
We've been naming conversions in ISLE like `foo_to_bar` since they aren't namespaced methods (so if we wanted to construct something else from `IntCC` the function name as written would be ambiguous).

So this should be `intcc_to_cc`.